### PR TITLE
Devise branch

### PR DESCRIPTION
**Pull request by Josphat205**
### **In This Branch**
Removed current_user method in ApplicationController, devise will provide us with one.
Installed and setup devise
- Can register a new user.
- User logs in with a combination of email and password.
- Hashed password should be stored in the database.
- Ask for confirmation of email.
- Can reset passworpasswordsd.
Add styling to match wireframe

### CODE REVIEW

File: app/views/devise/sessions/new.html.erb [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+      </div>
+    <% end %>
+  
+    <div class="actions">
+      <%= f.submit "Log in", class: 'btn-sm btn btn-success mt-2' %>
+    </div>
+  <% end %>
+  <%= render "devise/shared/links" %>
+</div>
+
```
Comment (CollinsTatang) [inline app/views/devise/sessions/new.html.erb]: Hi @Josphat205 
Nice job so far implementing the different project requirements but just some little adjustments to completely meet up the project requirements.

- [ ] Please modified your devise `login` and `registration` view file to match the wireframes and their styling.
image

File: README.md [hunk 2]
```diff
@@ -170,31 +152,35 @@ Contributions, issues, and feature requests are welcome!
...
 
 
 <!-- LICENSE -->
 
 ## 📝 License <a name="license"></a>
 
-This project is [MIT](LICENSE.txt) licensed.
-
-_NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
+This project is [MIT](./LICENSE) licensed.
```
Comment (CollinsTatang) [inline README.md]: - [ ] Please fix the link to your `MIT License` file the current link is broken.

**Review by CollinsTatang — CHANGES_REQUESTED** [review]
Hi @Josphat205, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

### **Highlights 🚀 :heavy_check_mark: :heavy_check_mark:** 

- The repo looks professional.:heavy_check_mark:
- Git/GitHub flow best practices have been followed:heavy_check_mark:
- There are no linter errors:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:

## Required changes 
It's ok to have bugs in your code. Please, see this code review as a chance for you to learn practical coding skills and learn new tips that can improve your code quality and help you get a better job.
_Check the comments under the review._
## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @CollinsTatang in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by prolajumokeoni — APPROVED** [review]
Hi @Josphat205 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
## Highlights
 - All changes requested have been implemented 🥇  
 - Design matches wireframe design ✔️

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**